src/query_results.js

```diff
@@ -204,221 +204,218 @@ module.exports = class QueryResult {
-    // but it's formatted differently for easier pre-processing.
-    const preresults = [];
-    this._getPreresults(
-      dataByEdge,
-      initialQueryEdgeID,
-      edgeCount,
-      preresults,
-      [], // first preresult
-      initialQueryNodeIDToMatch,
+    debug(`initialQEdgeID: ${initialQEdgeID}, initialQNodeIDToMatch: ${initialQNodeIDToMatch}`);
+
+    const queryGraphSolutions = [];
+    this._getQueryGraphSolutions(
+      recordsByQEdgeID,
+      initialQEdgeID,
+      qEdgeCount,
+      queryGraphSolutions,
+      [], // first queryGraphSolution
+      initialQNodeIDToMatch,
     );
 
     /**
      * Consolidation
      *
      * With reference to this graphic:
      * https://github.com/biothings/BioThings_Explorer_TRAPI/issues/341#issuecomment-972140186
-     * The preresults are analogous to the collection of sets in the lower left. Now we want
-     * to consolidate the preresults as indicated by the the large blue arrow in the graphic
-     * to get consolidatedPreresults, which are almost identical the the final results, except
+     * The queryGraphSolutions are analogous to the collection of sets in the lower left. Now we want
+     * to consolidate the queryGraphSolutions as indicated by the the large blue arrow in the graphic
+     * to get consolidatedSolutions, which are almost identical the the final results, except
      * for some minor differences that make it easier to perform the consolidation.
      *
-     * There are two cases where we need to consolidate preresults:
+     * There are two cases where we need to consolidate queryGraphSolutions:
      * 1. one or more query nodes have an 'is_set' param
-     * 2. one or more primaryID pairs have multiple kgEdges each
+     * 2. one or more primaryCurie pairs have multiple kgEdges each
      *
-     * We perform consolidation by first grouping preresults by uniqueResultID and
-     * then merging each of those groups into a single consolidatedPreresult.
+     * We perform consolidation by first grouping queryGraphSolutions by trapiResultID and
+     * then merging each of those groups into a single consolidatedSolution.
      */
 
-    const preresultsByUniqueResultID = {};
-    preresults.forEach((preresult) => {
-      // example inputPrimaryID and outputPrimaryID in a preresult:
+    const solutionsByTrapiResultID = {};
+    queryGraphSolutions.forEach((queryGraphSolution) => {
+      // example inputPrimaryCurie and outputPrimaryCurie in a queryGraphSolution:
       // [
-      //   {"inputPrimaryID": "NCBIGene:3630", "outputPrimaryID", "MONDO:0005068"},
-      //   {"inputPrimaryID": "MONDO:0005068", "outputPrimaryID", "PUBCHEM.COMPOUND:43815"}
+      //   {"inputPrimaryCurie": "NCBIGene:3630", "outputPrimaryCurie", "MONDO:0005068"},
+      //   {"inputPrimaryCurie": "MONDO:0005068", "outputPrimaryCurie", "PUBCHEM.COMPOUND:43815"}
       // ]
       //
       // Other items present in a presult but not shown above:
```

Comment:
Sorry, this is picky and it was my own typo, but `presult` -> `queryGraphSolution`